app/graphql/types/query_type.rb

```diff
@@ -37,5 +40,46 @@ def news_item(id:)
     def tour(id:)
       Tour.find(id)
     end
+
+    # Provide contents from html files in `public/mobile-app/contents` through GraphQL query
+    #
+    # @param [String] name the file name
+    #
+    # @return [String] the contents of the file, if it exists - otherwise ""
+    def public_html_file(name:)
+      public_contents_folder = "#{Rails.root}/public/mobile-app/contents/"
```

Comment:
ich würde eher den finalen "/" hier weglassen und unten in file="" integrieren

`file = "#{public_contents_folder}/#{name}.#{file_type}"`

sieht verständlicher aus

Reply:
mag eigentlich auch keine abschließenden `/`. hier macht es aber deutlich, dass der string auch wirklich ein Ordner ist. und fand, dass der unten übersehen werden könnte.
so müssen lediglich die variablen verkettet werden. auch den `.` fand ich am ende nicht ideal im string.

Reply:
habe es jetzt optimiert und mich hierfür entschieden:

```
public_contents_folder = "#{Rails.root}/public/mobile-app/contents"
file_type = "html"
file = File.join(public_contents_folder, "#{name}.#{file_type}")
```
damit erübrigt sich das `/` :)
außerdem wird unten ja bereits `queryable_files = File.join(path, "*.#{file_type}")` verwendet, so herrscht hier saubere einheitlichkeit!